docs/chapter-1/vulkan_renderpass_code.md

```diff
@@ -72,8 +72,8 @@ void VulkanEngine::init_default_renderpass()
 	// we keep the attachment stored when the renderpass ends
 	color_attachment.storeOp = VK_ATTACHMENT_STORE_OP_STORE;
 	//we don't care about stencil
-	color_attachment.stencilLoadOp = VK_ATTACHMENT_LOAD_OP_DONT_CARE;	
-	color_attachment.stencilStoreOp = VK_ATTACHMENT_STORE_OP_DONT_CARE;
+	color_attachment.stencilLoadOp = VK_ATTACHMENT_LOAD_OP_don't_CARE;	
```

Comment:
If this is due to a search and replace gone wrong, then it needs to be corrected.
Even the Vulkan API gets it 'wrong' it seems. (Though I would not want to have to type grammatically correct English for what should be a programming API...)